README.md

```diff
@@ -9,6 +9,7 @@ The IBM Power Virtual Systems Container Storage Interface (CSI) Driver provides
 | PowerVS CSI Driver \ CSI Version | v1.5.0 |
 | ----------------------------- | -------|
 | main branch | yes |
+| v0.1.0-alpha.2 | yes |
```

Comment:
we shouldn't be documenting this because this a pre-released version and only for the testing purpose, once we have proper release we can add the version

Reply:
fixed